Sources/MapboxCoreNavigation/NativeHandlersFactory.swift

```diff
@@ -102,14 +102,20 @@ class NativeHandlersFactory {
     }()
     
     static var navigatorConfig: NavigatorConfig {
+        var nativeIncidentsOptions: MapboxNavigationNative.IncidentsOptions?
+        if let incidentsOptions = NavigationSettings.shared.liveIncidentsOptions,
+           !incidentsOptions.graph.isEmpty || incidentsOptions.apiURL != nil {
```

Comment:
NN has [its own logic](https://github.com/mapbox/mapbox-navigation-native/blob/a8a7dc26f468a39275765ef41357745c6c546a97/src/mapbox/navigation/config.cpp#L178-L189) on these empty fields. It looks like an empty `graph` is always a mistake, but an absent `apiURL` is an expected value.

I think we should either remove this line to allow for states that NN allows (`graph` is empty, but incidents are enabled) and not depend on their implementation changes, or remove `apiURL` check to prohibit invalid case (incidents are on when `graph` is empty, but `apiURL` is set).

Reply:
Provided snippet just warns about the empty `graph` value, but still enables the incidents. So I cannot say that it is an error to do so.
This mirrors Android [implementation](https://github.com/mapbox/mapbox-navigation-android/blob/cbb01d24630d444198bd4f19f8435f395d22ad5f/libnavigation-core/src/main/java/com/mapbox/navigation/core/MapboxNavigation.kt#L313) which is important atm, but we could address this on both platforms as a tailwork.

Reply:
Got it. Please create a ticket to sync with Android and make this right, possibly before the release.